app/services/api/v1x0/catalog/approval_transition.rb

```diff
@@ -86,6 +86,11 @@ def canceled?
         def error?
           @approvals.present? && @approvals.any? { |req| req.state == "error" }
         end
+
+        def reasons
+          return "" unless @approvals.present?
+          @approvals.collect(&:reason).compact.join('.')
```

Comment:
Do you want a space between the `reason` text being joined here?  `join('. ')`

Reply:
@gmcculloug Done.